homeassistant/components/sensor/command_line.py

```diff
@@ -53,10 +53,9 @@ def __init__(self, hass, data, name, unit_of_measurement, value_template):
         self._hass = hass
         self.data = data
         self._name = name
-        self._state = STATE_UNKNOWN
+        self._state = None
```

Comment:
It's the base `Entity` class that should handle converting `None` to `STATE_UNKNOWN`.